docs/modules/getting_started/pages/installation.adoc

```diff
@@ -0,0 +1,62 @@
+= Installation
+
+On this page you will install the Stackable Spark-on-Kubernetes operator as well as the Commons and Secret operators which are required by all Stackable operators.
+
+== Dependencies
+
+The Spark-k8s Operator does not install a cluster or a set of nodes, but rather a SparkApplication that creates a spark-submit job, delegating this to the Spark driver, which in turn spins up executors for the duration of the job. Spark job dependencies can thus vary in their scope and detail. In this guide we are going to keep things simple and look at executing a Spark job that has a minimum of dependencies.
```

Comment:
Well then, let's write it like that :) Here is a suggestion:

```suggestion
Spark applications almost always require dependencies like database drivers, REST api clients and many others. These dependencies must be available on the `classpath` of executors and sometimes drivers. There are multiple ways to provision Spark jobs with such dependencies, some are built into Spark it's self while others are implemented at the operator level. In this guide we are going to keep things simple and look at executing a Spark job that has a minimum of dependencies.
```

Reply:
Thanks - much better! (just made a couple of minor tweaks to this)